$NetBSD: patch-aq,v 1.1 2007/03/06 13:08:24 rillig Exp $

Don't call varargs macros with zero arguments, since that is not
required by ISO C99 and only gcc can handle it.

Fixed in gettext-tools-0.16.

--- gettext-tools/src/po-lex.h.orig	Fri May 20 21:03:44 2005
+++ gettext-tools/src/po-lex.h
@@ -73,7 +73,7 @@ extern void po_lex_pass_obsolete_entries (bool flag);
 
 /* ISO C 99 is smart enough to allow optimizations like this.
    Note: OpenVMS 7.3 cc pretends to support ISO C 99 but chokes on '...'.  */
-#if __STDC__ && (defined __STDC_VERSION__ && __STDC_VERSION__ >= 199901L && !defined __DECC)
+#if 0
 
 /* CAUTION: If you change this macro, you must also make identical
    changes to the function of the same name in src/po-lex.c  */
@@ -109,7 +109,7 @@ extern void po_lex_pass_obsolete_entries (bool flag);
   } while (0)
 
 /* GCC is also smart enough to allow optimizations like this.  */
-#elif __STDC__ && defined __GNUC__ && __GNUC__ >= 2 && !defined __APPLE_CC__
+#elif 0
 
 /* CAUTION: If you change this macro, you must also make identical
    changes to the function of the same name in src/po-lex.c  */
